src/app/card/basic-card/examples/card-trend-example.component.ts

```diff
@@ -16,38 +18,38 @@ import { CardFilter } from '../../card-filter/card-filter';
 export class CardTrendExampleComponent implements OnInit {
   actionsText: string = '';
   chartDates: any[] = ['dates'];
-  chartConfigVirtual: any = {
+  chartConfigVirtual: SparklineConfig = {
     chartHeight: 60,
     chartId: 'virtualTrendsChart',
     tooltipType: 'default'
   };
-  chartDataVirtual: any = {
+  chartDataVirtual: SparklineData = {
     dataAvailable: true,
     total: 250,
     xData: this.chartDates,
     yData: [
       'used', '90', '20', '30', '20', '20', '10', '14', '20', '25',
       '68', '44', '56', '78', '56', '67', '88', '76', '65', '87', '76']
   };
-  chartConfigPhysical: any = {
```

Comment:
seems odd to use a sparkline config/data in here instead of something related to a trend view

Reply:
I believe this is just temporary because we don't have any other chart to show right now.